ktoml-core/src/iosMain/kotlin/com/akuleshov7/ktoml/utils/UtilsIos.kt

```diff
@@ -0,0 +1,11 @@
+package com.akuleshov7.ktoml.utils
+
+@Suppress("MAGIC_NUMBER")
+internal actual fun StringBuilder.appendCodePointCompat(codePoint: Int): StringBuilder = when (codePoint) {
+    in 0 until Char.MIN_SUPPLEMENTARY_CODE_POINT -> append(codePoint.toChar())
+    in Char.MIN_SUPPLEMENTARY_CODE_POINT..Char.MAX_CODE_POINT -> {
+        append(Char.MIN_HIGH_SURROGATE + ((codePoint - 0x10000) shr 10))
+        append(Char.MIN_LOW_SURROGATE + (codePoint and 0x3ff))
+    }
+    else -> throw IllegalArgumentException()
+}
```

Comment:
I just copied this code from the linux version, is there anything platform specific needed?

Reply:
Nope, I do not even think that this method is really a cross-platform code and cannot be in common :)